clar2wasm/src/error_mapping.rs

```diff
@@ -156,6 +160,36 @@ fn from_runtime_error_code(
             RuntimeErrorType::Arithmetic(POW_ERROR_MESSAGE.into()),
             Some(Vec::new()),
         ),
+        ErrorMap::NameAlreadyUsed => {
+            let runtime_error_arg_offset = instance
+                .get_global(&mut store, "runtime-error-arg-offset")
+                .and_then(|glob| glob.get(&mut store).i32())
+                .unwrap_or_else(|| {
+                    panic!("Could not find $runtime-error-arg-offset global with i32 value")
+                });
+
+            let runtime_error_arg_len = instance
+                .get_global(&mut store, "runtime-error-arg-len")
+                .and_then(|glob| glob.get(&mut store).i32())
+                .unwrap_or_else(|| {
+                    panic!("Could not find $runtime-error-arg-len global with i32 value")
+                });
+
+            let arg_name = String::from_utf8(
+                read_bytes_from_wasm(
+                    instance
+                        .get_memory(&mut store, "memory")
+                        .unwrap_or_else(|| panic!("Could not find wasm instance memory")),
+                    &mut store,
+                    runtime_error_arg_offset,
+                    runtime_error_arg_len,
+                )
+                .unwrap_or_else(|e| panic!("Could not read from wasm memory: {e}")),
+            )
+            .unwrap_or_else(|e| panic!("Could not recover arg_name: {e}"));
```

Comment:
We have a `read_identifier_from_wasm` [here](https://github.com/stacks-network/clarity-wasm/blob/9d5dd14627da885a4279aa0a092cb4c9597232ec/clar2wasm/src/wasm_utils.rs#L1164-L1172)